blocks/hero/hero.js

```diff
@@ -1,6 +1,12 @@
+/* eslint-disable max-len */
```

Comment:
let's don't disable rules for whole files if possible and reduce amount of exceptions.

Reply:
removed the exception.